lib/shims/eq/R2015b/contains.m

```diff
@@ -0,0 +1,9 @@
+function result = contains(str, pat)
+
+% CONTAINS Determine if pattern is in strings
+% CONTAINS Introduced in R2016b
+% https://www.mathworks.com/help/matlab/ref/contains.html
+
+result = length(strfind(str, pat)) > 0;
```

Comment:
I think you want `numel` instead of `length` here, to handle the case of non-vector inputs. Or `~isempty(x)`, which is equivalent to `numel(x) > 0`.

Reply:
I like your ~isempty() suggestion.  The PR has been updated with that.